### ISSUE TITLE
Client/Host crash when using schedules or moving duplicants there

### ISSUE THREAD

**Issue by MG665** [onimp/oni_multiplayer#145]
Crash occurs when setting up schedules, mainly moving duplicants to a different spot/menu.

```
NullReferenceException: Object reference not set to an instance of an object

Schedule.Changed () (at <cd4b9bd5aa6c4ec38ec00dca1dc79105>:0)
MultiplayerMod.Multiplayer.Commands.Screens.Schedule.ChangeSchedulesList.Execute () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Multiplayer.Configuration.MultiplayerCoordinator+<>c__DisplayClass12_0.<ClientOnCommandReceived>b__1 () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl+<>c__DisplayClass1_0.<RunWithDisabledPatches>b__0 () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl.RunWithDisabledPatches[T] (System.Func`1[TResult] action) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl.RunWithDisabledPatches (System.Action action) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Multiplayer.Configuration.MultiplayerCoordinator.ClientOnCommandReceived (System.Object sender, MultiplayerMod.Network.Events.CommandReceivedEventArgs e) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.SteamClient.ReceiveCommands () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.SteamClient.Tick () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.Components.SteamClientComponent.Update () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)

Build: U47-562984-V
```

**Comment by MG665** [comment]
Same issue happens when printing new duplicants by the client (skipping host printing since they keep getting the same duplicant choices):

```
NullReferenceException: Object reference not set to an instance of an object

Schedule.Changed () (at <cd4b9bd5aa6c4ec38ec00dca1dc79105>:0)
MultiplayerMod.Multiplayer.Commands.Screens.Schedule.ChangeSchedulesList.Execute () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Multiplayer.Configuration.MultiplayerCoordinator+<>c__DisplayClass12_0.<ClientOnCommandReceived>b__1 () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl+<>c__DisplayClass1_0.<RunWithDisabledPatches>b__0 () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl.RunWithDisabledPatches[T] (System.Func`1[TResult] action) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl.RunWithDisabledPatches (System.Action action) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Multiplayer.Configuration.MultiplayerCoordinator.ClientOnCommandReceived (System.Object sender, MultiplayerMod.Network.Events.CommandReceivedEventArgs e) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.SteamClient.ReceiveCommands () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.SteamClient.Tick () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.Components.SteamClientComponent.Update () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)

Build: U47-562984-V
```

**Comment by zuev93** [comment]
Hi @MG665 
Thanks for the report!

It seems to be the same kind of issue which appears only for objects that appears after game sync (e.g. newly printed duplicants).
We haven't implemented sync of their ids and usally interaction with them is errorprone.

I've fixed the client failure and it'll write to the logs instead.
However schedule for those duplicants wont be fixed until we sync their Ids (see #132 and #133)
